### PR TITLE
docs(root): update readme of all nx plugins with template

### PR DESCRIPTION
**Pull request by KarimHmaissi**
update readme of all nx plugins with template

fix #5392

### CODE REVIEW

File: packages/azure-node/README.md [hunk 1]
```diff
@@ -1,12 +1,55 @@
...
+## Generators and Executors
 
-Run `nx test azure-node` to execute the unit tests via
-[Jest](https://jestjs.io).
+`plugin-name` contains the following generators and executors
+
+```bash
+nx g @ensono-stacks/[plugin-name]:[generator-executor-name]
+```
+
```
Comment (Danphillipz) [inline packages/azure-node/README.md:17]:
```suggestion
View a list of the plugin executors/generators through the following command:

```bash
nx list @ensono-stacks/azure-node
```

```

File: packages/azure-node/README.md [hunk 2]
```diff
@@ -1,12 +1,55 @@
...
+Run the following to publish the NPM package
+
+```bash
+nx publish plugin-name
+```
+
+## Full documentation
+
+Please visit the stacks documentation page for `plugin-name`
+[here](http://add-link-here-to-documentation) for more information
```
Comment (Danphillipz) [inline packages/azure-node/README.md:55]:
```suggestion
[here](https://stacks.amido.com/docs) for more information
```

**Review by Danphillipz — CHANGES_REQUESTED** [review]
I think for each plugin we have:
1. Apply the suggested change referencing the stacks docs (not a 1-1 link yet, but good for now)
2. Swap plugin-name to the actual name of each plugin for each readme file

Then at least we have valid ReadMe files for each plugin, we can come back at a later date to add the 1-1 link to docs, but for now, having it just point of our existing docs site should suffice

File: packages/azure-node/README.md (hunk 1 above)
Comment (KarimHmaissi) [reply packages/azure-node/README.md]: Good shout. Updated

File: packages/azure-node/README.md (hunk 2 above)
Comment (KarimHmaissi) [reply packages/azure-node/README.md]: Updated

**Review by Danphillipz — APPROVED** [review]
LGTM

**Pull request merged**